starknet_devnet/starknet_wrapper.py

```diff
@@ -215,6 +212,7 @@ async def __create_genesis_block(self):
             deploy_data.append((account.class_hash, account.address))
 
         for class_hash, contract_address in deploy_data:
+            # this might be the only place where DEPLOY tx is used
             internal_deploy = create_empty_internal_deploy(
```

Comment:
Ok but this will remain for now and will be removed in the next iteration or there is no plan to remove it?

Reply:
I think it can stay like this until the cairo-lang package completely drops this transaction type.